taskDev7/src/dateFormatChecker/Validator.java

```diff
@@ -0,0 +1,91 @@
+package dateFormatChecker;
+
+import dateFormatChecker.dateChecker.DateChecker;
+import dateFormatChecker.dateChecker.DayOfMonthChecker;
+import dateFormatChecker.dateChecker.HourChecker;
+import dateFormatChecker.dateChecker.MinuteChecker;
+import dateFormatChecker.dateChecker.MonthChecker;
+import dateFormatChecker.dateChecker.YearChecker;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Checks that input string satisfies fixed format.
+ */
+public class Validator {
+
+  /**
+   * Checks that input string is date in fixed format.
+   *
+   * @param inputString given date.
+   * @return true if it satisfies format.
+   */
+  public boolean isValid(String inputString) {
+    boolean isValidValue = false;
+    List<DateChecker> checkerArray = Arrays.asList(new DayOfMonthChecker(), new MonthChecker(),
```

Comment:
it must be chain of responsibility, not just array of checkers as it's too easy to make error if you pass date in other order